### ISSUE TITLE
No error when scaffold source doesn't exist and prestissimo installed globally

### ISSUE THREAD

**Issue by datvance** [drupal-composer/drupal-scaffold#73]
Not sure if this is an issue with drupal-scaffold or prestissimo, but if you have prestissimo installed globally, as per their docs, no errors or exceptions are thrown if the scaffold source cannot be found.

```json
"drupal-scaffold": {
      "source": "https://raw.example.com/pantheon-systems/drops-8/{version}/{path}",
```
Should generate an error since example.com does not exist, but no errors:
```
$ composer install
...
Writing lock file
Generating optimized autoload files
$ cat web/.gitattributes
cat: web/.gitattributes: No such file or directory
```
After removing prestissimo, composer install correctly throws errors:
```
$ composer global remove hirak/prestissimo
$ composer install
...
Generating optimized autoload files
Downloading (failed)       
                                                                                                                                                                         
  [Composer\Downloader\TransportException]                                                                                                                               
  The "https://raw.example.com/pantheon-systems/drops-8/8.4.5/.gitattributes" file could not be downloaded: php_network_getaddresses: getaddrinfo failed: nodename nor   
  servname provided, or not known                                                                                                                                        
  failed to open stream: php_network_getaddresses: getaddrinfo failed: nodename nor servname provided, or not known                                                      
                                                                                                                                                                         
Exception trace:
...
$ cat web/.gitattributes
cat: web/.gitattributes: No such file or directory
```
Like I said, if this is a prestissimo issue I can submit it over there. Thanks.

**Comment by FlorentTorregrosa** [comment]
Hi @datvance,

I have been able to reproduce the bug.

I tried to wrap the content of the method into a try / catch to see if we can react to an exception. But no exception got catched.

```php
    try {
      foreach ($this->filenames as $sourceFilename => $filename) {
...
      } while ($multi->remain());
    }
    catch (\Exception $e) {
      $this->io->writeError($e->getMessage());
    }
```

So yeah maybe it is Prestissimo that does not handle errors correctly so we cannot react appropriately.

Maybe we can wait for comments from drupal-scaffold plugin maintainers.
